### PR TITLE
Telephone capture CE unit individual fix

### PR DESCRIPTION
**Pull request by AdamHawtin**
# Motivation and Context
Telephone capture for CE Units should have the individual parameter set to receive an individual QID type back

# What has changed
* Move CE Unit telephone capture scenarios to individual
* Small refactor to makes tests around telephone capture clearer

# How to test?
Run with linked case API fix branch

# Links
https://trello.com/c/gKgbepbP/548-defect-telephone-capture-ce-unit-api-call-not-as-per-solution-3#
https://github.com/ONSdigital/census-rm-case-api/pull/58

### CODE REVIEW

**Comment by AdamHawtin** [comment]
[Defect: Telephone Capture CE unit API call not as per solution (3)](https://trello.com/c/gKgbepbP/548-defect-telephone-capture-ce-unit-api-call-not-as-per-solution-3)

File: acceptance_tests/features/steps/telephone_capture.py [hunk 1]
```diff
@@ -22,7 +22,21 @@ def request_telephone_capture_qid_uac(context, address_level, case_type, country
     context.telephone_capture_qid_uac = response.json()
 
 
-@step('there is a request for individual telephone capture for a unit case '
+@step('there is a request for individual telephone capture for the case '
+      'with case type "{case_type}" and country "{country_code}"')
+def spg_unit_individual_request(context, case_type, country_code):
```
Comment (liamtoozer) [inline acceptance_tests/features/steps/telephone_capture.py]: Minor comment but this handles CEs as well, small tweak to name of function maybe?

**Review by liamtoozer — APPROVED** [review]
LGTM 👍

**Review by NickGrantX — APPROVED** [review]
LGTM 👍

**Pull request merged**